src/Command/GenerateJsonCommand.php

```diff
@@ -119,13 +119,14 @@ private function generatePrestaShopModulesJson(
             }
         }
 
+        $modulesPath = $this->jsonDir . '/modules/';
+        if (!is_dir($modulesPath)) {
+            mkdir($modulesPath);
```

Comment:
Should the result get tested, in case of file system permission issue ?
```suggestion
            if (!mkdir($modulesPath)) {
                throw new WhateverException(sprintf('Failed to create directory "%s"', $modulesPath));
            }
```